.rubocop_todo.yml

```diff
@@ -203,11 +203,6 @@ Style/EmptyElse:
 Style/EmptyLineBetweenDefs:
   Enabled: false
 
-# Offense count: 142
-# Cop supports --auto-correct.
-Style/EmptyLines:
-  Enabled: false
-
 # Offense count: 20
 # Cop supports --auto-correct.
 Style/EmptyLinesAroundAccessModifier:
```

Comment:
Yes. The rubocop_todo is essentially a blacklist of lint rules. By removing it from the blacklist, it reverts to the default [whitelist].